lbry/file_analysis.py

```diff
@@ -14,18 +14,26 @@
 
 class VideoFileAnalyzer:
 
+    def _replace_or_pop_env(self, variable):
+        if variable + '_ORIG' in self._env_copy:
+            self._env_copy[variable] = self._env_copy[variable + '_ORIG']
+        else:
+            self._env_copy.pop(variable, None)
```

Comment:
What's the reason for poping the variable if there is no `_ORIG`? This seems like it would give you a different behavior during development vs. running a built binary since in development no LD_LIBRARY_PATH is forwarded at all but when run in binary you actually get the same LD_LIBRARY_PATH that would have been forwarded during development...

Reply:
That's what their instructions said to do. If there is no `_ORIG` value then there was not a LD_LIBRARY_PATH value existing before we started running. So if there still isn't one then the pop is a no-op, if there is one then it must have been added by the pyinstaller bootstrapper.

Reply:
There will be no `_ORIG` if lbrynet was run outside of pyinstaller boostrapper but yet there will still be a valid `LD_LIBRARY_PATH` that should be passed to the executor.

Reply:
Fixed.